client/jetpack-cloud/sections/agency-dashboard/sites-overview/index.tsx

```diff
@@ -229,25 +259,30 @@ export default function SitesOverview() {
 		);
 	};
 
-	const showIssueLicenseButtonsLargeScreen =
-		isWithinBreakpoint( '>960px' ) && selectedLicensesCount > 0;
-
 	return (
 		<div className="sites-overview">
 			<DocumentHead title={ pageTitle } />
 			<SidebarNavigation sectionTitle={ pageTitle } />
 			<div className="sites-overview__container">
+				{ isWithinBreakpoint( '<960px' ) &&
+					selectedLicensesCount < 1 &&
+					renderAddSiteIssueLicenseButtons() }
 				<div className="sites-overview__tabs">
 					<div className="sites-overview__content-wrapper">
 						<SiteSurveyBanner isDashboardView />
 						<SiteWelcomeBanner isDashboardView />
 						<SiteDowntimeMonitoringUpgradeBanner />
-
 						{ data?.sites && <SiteAddLicenseNotification /> }
 						<SiteContentHeader
-							content={ renderIssueLicenseButton() }
+							content={
+								// don't render content on mobile, we don't need it
+								selectedLicensesCount > 0 && isWithinBreakpoint( '>960px' )
+									? renderIssueLicenseButton()
+									: renderAddSiteIssueLicenseButtons()
+							}
 							pageTitle={ pageTitle }
-							showStickyContent={ !! showIssueLicenseButtonsLargeScreen }
+							// Only renderIssueLicenseButton should be sticky.
+							showStickyContent={ selectedLicensesCount > 0 }
```

Comment:
I think we need `isWithinBreakpoint( '>960px' )` check here.

Reply:
Yes indeed, that was causing double buttons some times.